docker-compose.yaml

```diff
@@ -1,56 +1,59 @@
-version: '2'
+version: '3.8'
```

Comment:
If docker-compose using version 3.8, the version on docker-compose installation must change to 1.25.5 in line 83 in README.md

```
sudo curl -L https://github.com/docker/compose/releases/download/1.25.5/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
```

Reply:
Yes indeed. I'll fix right away.